app/containers/Header/ProfileDropdown.jsx

```diff
@@ -103,146 +103,154 @@ const B = ({ profileInfo, onClick, isMenuVisible, isMobileVersion }) => (
-            <FormattedMessage id={messages.profile.id} />
-          </A>
-
-          <A to={routes.userCommunities(user)}>
-            <CommunitiesIcon
-              className="mr-2"
-              size={[18, 18]}
-              stroke={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.myCommunities.id} />
-          </A>
-
-          <A
-            to={routes.userQuestions(user)}
-            disabled={!questionsLength}
-            tabIndex={!questionsLength ? '-1' : undefined}
-          >
-            <PostIcon
-              className="mr-2"
-              stroke={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.posts.id} />
-          </A>
-
-          <A
-            to={routes.userAnswers(user)}
-            disabled={!questionsWithUserAnswersLength}
-            tabIndex={!questionsWithUserAnswersLength ? '-1' : undefined}
-          >
-            <AnswerWithAIcon
-              className="mr-2"
-              stroke={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.answers.id} />
-          </A>
-
-          <A to={routes.userNotifications(user)}>
-            <NotificationsIcon
-              className="mr-2"
-              stroke={colors.linkColor || TEXT_PRIMARY}
-              fill={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.notifications.id} />
-          </A>
-
-          <A to={routes.userNFTs(user)}>
-            <NFTIcon
-              className="mr-2"
-              stroke={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.NFTs.id} />
-          </A>
+const Link = A.extend`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+`;
 
-          {isModerator && (
-            <A to={routes.userModeration(user)}>
-              <ModerationIcon
-                size={[18, 18]}
-                className="mr-2"
-                stroke={colors.linkColor || TEXT_PRIMARY}
-                fill={colors.linkColor || TEXT_PRIMARY}
-              />
-              <FormattedMessage id={messages.moderation.id} />
-            </A>
-          )}
+const getOptions = (profileInfo) => {
+  const { user } = profileInfo;
 
-          <A to={routes.userSettings(user)}>
-            <SettingsIcon
-              className="mr-2"
-              stroke={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.settings.id} />
-          </A>
-        </Ul>
+  const isModerator = useMemo(
+    () => !!getPermissions(profileInfo)?.length,
+    [profileInfo],
+  );
 
-        <Ul>
-          <Logout>
-            <LogOutIcon
-              className="mr-2"
-              stroke={colors.linkColor || TEXT_PRIMARY}
-            />
-            <FormattedMessage id={messages.logout.id} />
-          </Logout>
-        </Ul>
-      </nav>
-    );
-  },
-);
+  return [
```

Comment:
put in a separate config in a separate file

Reply:
done